packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/ReactNativeSessionReplayExtensionSupportTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.datadog.reactnative.sessionreplay
+
+import com.datadog.android.sessionreplay.SessionReplayPrivacy
+import com.datadog.reactnative.sessionreplay.mappers.ReactTextMapper
+import com.datadog.reactnative.sessionreplay.mappers.ReactViewGroupMapper
+import com.datadog.tools.unit.GenericAssert.Companion.assertThat
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.uimanager.UIManagerModule
+import com.facebook.react.views.text.ReactTextView
+import com.facebook.react.views.textinput.ReactEditText
+import com.facebook.react.views.view.ReactViewGroup
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.kotlin.any
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.whenever
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+internal class ReactNativeSessionReplayExtensionSupportTest {
+
+    @Mock
+    lateinit var mockReactContext: ReactContext
+
+    @Mock
+    lateinit var mockUiManagerModule: UIManagerModule
+
+    private lateinit var testedExtensionSupport: ReactNativeSessionReplayExtensionSupport
+
+    @BeforeEach
+    fun `set up`() {
+        whenever(mockReactContext.getNativeModule(any<Class<NativeModule>>()))
+            .doReturn(mockUiManagerModule)
+
+        testedExtensionSupport = ReactNativeSessionReplayExtensionSupport(mockReactContext)
+    }
+
+    @Test
+    fun `M get custom view mappers W getCustomViewMappers()`() {
+        // When
+        val customViewMappers = testedExtensionSupport.getCustomViewMappers()
+        val allowMappers = customViewMappers.get(SessionReplayPrivacy.ALLOW)
```

Comment:
this way result won't be nullable.

```suggestion
        val allowMappers = customViewMappers.getValue(SessionReplayPrivacy.ALLOW)
```

alternatively in the assertion section you can call `check(allowMappers != null)`, this way smart cast for calls below will be applied.